guideline-app/app/ui/components/tabbar/Tabbar.js

```diff
@@ -69,7 +69,7 @@ const TabbarItem = (props) => {
             role="button"
             tabIndex={0}
         >
-            <EtikettLiten>{ props.label }</EtikettLiten>
+            <EtikettLiten><span dangerouslySetInnerHTML={{ __html: props.label }} /></EtikettLiten>
```

Comment:
Hvorfor trenger vi egentlig gjøre dette / sende inn "Retnings&shy;linjer" som label-prop til Tabbar?

Reply:
Måtte gjøre det slik for å kunne [legge inn `&shy;` i label-tekstene](https://github.com/navikt/nav-frontend-moduler/pull/196/files#diff-1bb3154d88d3bfd75adeb9c618a8147e), slik at de brekker pent på de minste skjermene :)

<img width="311" alt="screen shot 2018-01-31 at 08 32 14" src="https://user-images.githubusercontent.com/74510/35610363-479ba09e-0661-11e8-98b6-61a6a51357e4.png">